### DISCUSSION TITLE
WordPress plugin

### DISCUSSION THREAD

**Discussion by RougeTwo** [community/community#125714] [Programming Help] [answered]
### Body

Hello the web. I really need some help. I am trying to build an interactive periodic table of the elements as a wordpress plugin. When I run the file it creates a lay opt of the table with the element symbols no problem. then when an element is clicked a pop up window appears and displays more details about that element. the challenge is that no all the titles are being populated. the first four no hassel, but the last two are not for the love of money showing up. Any advise please. Attached .zip has PHP/html, CSS and JS.
[Interactive-periodic-table240525.zip](https://github.com/community/community/files/15444221/Interactive-periodic-table240525.zip)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ebndev** [comment]
Hi @RougeTwo, 

Not sure if you're still having this issue but by the last 2 titles titles, do you mean `type` and `density`?

If so, I don't see elements for those in the popup div in the HTML or variables / methods for them in the JS files.

**Reply by RougeTwo** [reply, marked as answer]
Hello @ebndev, excuse the reply coming so long. I did find a work around eventually. 
I did mean type and density. i tried every thing i could, but the clash was being caused by a plugin for the site theme. i pivoted and created links to pages for the individual elements housed on independent pages. By far the least elegant solution, but it is working. Please see at www.wtiwleer.co.za, thank you so much for reaching out.

**Reply by RougeTwo** [reply]
https://www.wtiwleer.co.za/die-periodieke-tabel/

**Reply by ebndev** [reply]
@RougeTwo I'm glad you were able to resolve it! I'm going to close this Discussion then. 😄 

Thanks for being a part of the GitHub Community!
